pkgs/applications/graphics/qvge/default.nix

```diff
@@ -25,15 +27,15 @@ mkDerivation rec {
     inherit graphviz;
   });
 
-  nativeBuildInputs = [ qmake ];
+  nativeBuildInputs = [ wrapQtAppsHook qmake ];
 
-  buildInputs = [ qtx11extras ];
+  buildInputs = [ qtx11extras ] ++ lib.optional stdenv.isDarwin qtsvg;
```

Comment:
Is `qtx11extras` really required on darwin?

Reply:
Thank you for catching this, fixed now